### PR TITLE
Switch back to sqlite

### PR DESCRIPTION
**Pull request by wilwade**
Simple switch back to sqlite before we deploy the backend.

### CODE REVIEW

File: src/networkData.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ const frequencyRococo: NetworkData = {
   dripAmount: "5000",
   explorer: null,
   networkName: "Frequency Rococo Testnet",
-  rpcEndpoint: "wss://rpc.rococo.frequency.xyz",
+  // rpcEndpoint: "wss://rpc.rococo.frequency.xyz",
+  rpcEndpoint: "ws://localhost:9944",
```
Comment (enddynayn) [inline src/networkData.ts:49]: accident?

Comment (wilwade) [reply src/networkData.ts:49]: Yes. Thanks.

**Review by aramikm — COMMENTED** [review]
The code Looks good. I had issues when tried the UI locally and saw this error. Might be my local config

Screenshot 2024-04-01 at 4 37 43 PM

**Review by aramikm — APPROVED** [review]
Approving to unblock!

**Pull request merged**